rllib/connectors/tests/test_agent.py

```diff
@@ -190,6 +194,228 @@ def test_view_requirement_connector(self):
         self.assertTrue("prev_actions" in processed[0].data.for_action)
 
 
+@pytest.mark.skip(reason="activate when view_requirement is fully implemented.")
```

Comment:
do you think we should add these tests with the next PR?
so they can get reviewed together?

Reply:
I don't know how to take this part out of the current PR. I have the other PR going on in parallel and it is in sync with this PR. if I take this part out and then merge it will remove it from the other one too. There is no need to review it since it is skipped in pytest. We'll review it once we unskip it.